sdk/communication/communication-rooms/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Azure client library for Azure Communication Rooms services",
   "version": "1.0.0-beta.2",
   "engines": {
-    "node": ">=14.0.0"
+    "node": ">=12.0.0"
```

Comment:
What is the reasoning behind downgrading node version ?

Reply:
one of the language tools ( lint ) did that.

Reply:
Anuj, can we please recheck this. I checked few communication package.json, that were recently changed, and the node version is 14.
Can we locally, stash this change and run the language tools again.

Reply:
Undid the changes

Reply:
Thank you.